<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Amazon Dropbox Log Parser</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="input_Browse">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="log_Parse">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Parse Logs</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_Path">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>591</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Input Path</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="output_Path">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>591</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Output Path</string>
    </property>
   </widget>
   <widget class="QPushButton" name="output_Browse">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
